src/client/public/locale/en/common.json

```
@@ -413,26 +413,26 @@
   "hours": "hours",
   "minute": "minute",
   "minutes": "minutes",
-  "waterIngressRequiredFieldsAlert": "'Quantity', 'Start time' and 'Reliability of data' are mandatory fields, the entry cannot be saved if they are not filled out.",
+  "waterIngressRequiredFieldsAlert": "'Quantity', 'Start' and 'Reliability of data' are mandatory fields, the entry cannot be saved if they are not filled out.",
   "hydrogeology": "hydrogeology",
-  "groundwater_level_measurement": "Groundwater level measurement",
-  "groundwater_level_measurement_word_break": "Groundwater level measurement",
-  "gwlm_kind": "Type of groundwater measurement",
-  "gwlm_levelm": "Level (m)",
-  "gwlm_levelmasl": "Level (m a.s.l.)",
-  "gwlmRequiredFieldsAlert": "'Type of groundwater measurement', 'Start time' and 'Reliability of data' are mandatory fields, the entry cannot be saved if they are not filled out.",
-  "msgGroundwaterLevelMeasurementsEmpty": "No groundwater level measurements available.",
+  "groundwater_level_measurement": "Groundwater measurement",
+  "groundwater_level_measurement_word_break": "Groundwater measurement",
+  "gwlm_kind": "Method of groundwater measurement",
+  "gwlm_levelm": "Depth [m MD]",
+  "gwlm_levelmasl": "Level [m.a.s.l.]",
+  "gwlmRequiredFieldsAlert": "'Method of groundwater measurement', 'Start' and 'Reliability of data' are mandatory fields, the entry cannot be saved if they are not filled out.",
+  "msgGroundwaterLevelMeasurementsEmpty": "No groundwater measurements available.",
   "hydrotest": "Hydrotest",
-  "hydrotestKind": "Hydrotest kind",
+  "hydrotestKind": "Typ of hydrotest",
```

@gysimichael Stimmt diese Bezeichnung oder sollte es eher `Type` anstatt `Typ` heissen?
```suggestion
  "hydrotestKind": "Type of hydrotest",
```